app/views/components/gds/Styles.scala

```diff
@@ -19,6 +19,8 @@ package views.components.gds
 object Styles {
 
   val gdsPageLegend = "govuk-fieldset__legend--l"
+  val gdsPageLegendLarge = "govuk-fieldset__legend--xl"
```

Comment:
Why are we adding "large" styles?   The plan was to change the definitions from "...l" to "...xl" at some point

Reply:
Wasn't aware of that. '--l' did not match the prototype so I created those.
So should it be just '--l' ?

Reply:
Yes.  That is the reason for creating the constants - so we can switch to the "larger" styles en-mass when all pages are GDS